maintainers/maintainer-list.nix

```diff
@@ -5679,6 +5679,11 @@
     githubId = 1365692;
     name = "Will Fancher";
   };
+  elysasrc = {
+    name = "Elysa";
```

Comment:
Would you mind adding an email or matrix?

Reply:
Added my email.